scripts/create_server_conf.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# exit early in case the file already exists
+if [ -f /home/suwayomi/.local/share/Tachidesk/server.conf ]; then
```

Comment:
This means the file will not get updated even if you change env-vars, which feels sort of confusing.

Reply:
It will, this file just creates the config for the first time, its not used after that